main.tf

```diff
@@ -290,3 +292,44 @@ resource "google_storage_bucket_iam_member" "api_s_account_storage_admin_on_proj
 
   depends_on = ["google_project_service.project_services"]
 }
+
+/******************************************
+  compute.networkUser role granted to API and GKE service accounts for GKE on shared VPC 
+ *****************************************/
+resource "google_project_iam_binding" "gke_shared_vpc_project" {
+  count = "${(var.shared_vpc != "" && contains(var.activate_apis, "container.googleapis.com") && length(compact(var.shared_vpc_subnets)) == 0) ? 1 : 0}"
```

Comment:
Let's consolidate `var.shared_vpc != "" && contains(var.activate_apis, "container.googleapis.com")` in a single local boolean test at the top.

Reply:
ah yes, makes a ton of sense